src/interfaces/optional-readonly-props.ts

```diff
@@ -0,0 +1,38 @@
+// Optional and readonly properties in interfaces
+
+// Interfaces in TypeScript offer the flexibility of defining optional and readonly properties.
```

Comment:
we can write here that optional fields are designated using (?) operator and remove the sentence of Optional Properties alltogether